tests/lib/rules/no-string-prototype-extensions.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+const rule = require('../../../lib/rules/no-string-prototype-extensions');
+const RuleTester = require('eslint').RuleTester;
+
+const { ERROR_MESSAGE } = rule;
+
+const ruleTester = new RuleTester({
+  parser: require.resolve('babel-eslint'),
+  parserOptions: { ecmaVersion: 6, sourceType: 'module' },
+});
+
+ruleTester.run('no-string-prototype-extensions', rule, {
+  valid: [
+    "dasherize('myString')",
+    'capitalize(someString)',
+    "htmlSafe('<b>foo</b>')",
+    "someString['foo']()",
+    'something.foo()',
+  ],
```

Comment:
Can you add `dasherize.foo()` as a valid example?

Reply:
done